cartofacade/capi.go

```diff
@@ -407,7 +407,7 @@ func toError(status C.int) error {
 	case C.VIAM_CARTO_SUCCESS:
 		return nil
 	case C.VIAM_CARTO_UNABLE_TO_ACQUIRE_LOCK:
-		return errors.New("VIAM_CARTO_UNABLE_TO_ACQUIRE_LOCK")
+		return ErrUnableToAcquireLock
```

Comment:
This is made public so that callers of the cartofacade go package can check if a given error is this one.